### DISCUSSION TITLE
PBPK models of Probenecid and Furosemide to Predict Transporter Mediated Drug-Drug Interactions

### DISCUSSION THREAD

**Discussion by HannahBri** [Open-Systems-Pharmacology/Forum#738] [Announcement]
Dear OSP-Community,

We have been working on models for transporter mediated DDI predictions. Two whole-body PBPK models of probenecid (clinical OAT inhibitor) and furosemide (clinical OAT substrate) were carefully developed and then evaluated by prediction of the probenecid-furosemide and probenecid-rifampicin DDIs.

The corresponding manuscript, including comprehensive documentation, has been published online in Pharmaceutical Research. The models as well as PK-Sim projects presenting the predicted DDI studies can be found [here]( https://github.com/Open-Systems-Pharmacology?language=&page=1&q=osp-model+pbpk+pk-sim+ddi&type=).

Related publication:
Britz H, Hanke N, Taub ME, Wang T, Prasad B, Fernandez É, Stopfer P, Nock V, Lehr T. Physiologically Based Pharmacokinetic Models of Probenecid and Furosemide to Predict Transporter Mediated Drug-Drug Interactions. Pharm Res. 2020 Nov 25;37(12):250. (https://doi.org/10.1007/s11095-020-02964-z)

Best regards,
Hannah

**Comment by tobiasK2001** [comment]
Super! Thanks for that :-)

**Comment by StephanSchaller** [comment]
Great! Thank you!

Is it planned to create qualification reports "GitHub-Style"?
